exchanges/gateio/gateio_test.go

```diff
@@ -3401,7 +3450,6 @@ func TestOrderbookChannelIntervals(t *testing.T) {
 			if exp != nil {
 				require.ErrorIs(t, err, exp)
 			} else {
-				require.NoError(t, err)
 				switch {
```

Comment:
🦍 This is part.2 from above.
```suggestion
				switch {
				case s.Channel == "unknown_channel":
					assert.Empty(t, i, "orderbookChannelInterval should return empty for unknown channels")
```